optimum/intel/openvino/training_args.py

```diff
@@ -0,0 +1,29 @@
+from dataclasses import dataclass, field
+
+from transformers import TrainingArguments
+
+
+@dataclass
+class OVTrainingArguments(TrainingArguments):
+    """
+    Arguments pertaining to OpenVINO/NNCF-enabled training flow
+    """
+
+    nncf_compression_config: str = field(
+        default=None, metadata={"help": "NNCF configuration .json file for compression-enabled training"}
+    )
+    teacher_model_or_path: str = field(
```

Comment:
I agree, @vuiseng9 can you please address this comment?

Reply:
Thanks for the feedback, the current design of `OVTrainingArguments` only extends `TrainingArguments` with distillation hyperparameters (weightage and temperature) since users are required to handle teacher loading and feeding it as an argument ti `OVTrainer`. Similarly, `nncf_compression_config` is removed as user is expected to set `OVConfig` to `OVTrainer`.